cmake/modules/FindCurl.cmake

```diff
@@ -38,6 +38,13 @@ if(${CURL_LIBRARY} MATCHES ".+\.a$" AND PC_CURL_STATIC_LDFLAGS)
   pkg_check_modules(PC_NGHTTP2 libnghttp2 QUIET)
   find_library(NGHTTP2_LIBRARY NAMES libnghttp2 nghttp2
                                HINTS ${PC_NGHTTP2_LIBDIR})
+
+  find_library(BROTLIDEC_LIBRARY NAMES brotlidec
+                                 HINTS ${PC_BROTLI_LIBDIR})
+  find_library(BROTLICOMMON_LIBRARY NAMES brotlicommon
+                                    HINTS ${PC_BROTLI_LIBDIR})
+  set(BROTLI_LIBRARIES ${BROTLIDEC_LIBRARY} ${BROTLICOMMON_LIBRARY})
```

Comment:
Id forgotten how a lot of our old stuff "discovered" libraries. This is the shitty way, but i have other work that redoes the curl find module completely, so as long as @wsnipex is ok with this in the interim.

The "new" way i'll be proposing will be https://github.com/fuzzard/xbmc/commit/2ce516d254401521145456fa44f66d36509745dc
It essentially moves to cmake for internal stuff (windows/android/apple), but has a fallback for using pkgconfig data for the additional libraries in the event of a static curl lib being found, therefore we dont need this manual explicit find library calls for nghttp2 and now brotli, we just rely on what the pkgconfig provides for additional link requirements